src/EditorFeatures/Core/Implementation/IntelliSense/AsyncCompletion/CompletionSource.cs

```diff
@@ -289,6 +289,12 @@ public async Task<object> GetDescriptionAsync(IAsyncCompletionSession session, V
             CompletionService completionService,
             Dictionary<string, AsyncCompletionData.CompletionFilter> filterCache)
         {
+            var importItem = roslynItem as TypeImportCompletionItem;
```

Comment:
instead of having this specific subtype, should we maybe just support .CachedObject at teh CompletionItem level?

Reply:
Fixed